src/radio-group/interfaces.ts

```diff
@@ -44,6 +44,12 @@ export interface RadioGroupProps extends BaseComponentProps, FormFieldControlPro
    * Called when the user selects a different radio button. The event `detail` contains the current `value`.
    */
   onChange?: NonCancelableEventHandler<RadioGroupProps.ChangeDetail>;
+
+  /**
+   * Deprecated, has no effect.
+   * @deprecated
```

Comment:
Shouldn't we follow the same writing style like here https://github.com/cloudscape-design/components/blob/main/src/internal/base-component/index.ts#L28

Reply:
It follows name in select https://github.com/cloudscape-design/components/blob/main/src/select/interfaces.ts#L85

Reply:
The one from select is shown on the website API tab and the one from the base component is not. So I think the select-like pattern is to be preferred and ideally it would be nice to update the base component's API description.